vk_video_decoder/libs/NvVideoParser/src/VulkanVP9Decoder.cpp

```diff
@@ -246,9 +246,9 @@ uint32_t VulkanVP9Decoder::UpdateForwardProbability(vp9_prob_update_s *pProbSetu
         pProbSetup->transform_mode = ONLY_4X4;
     else
     {
-        pProbSetup->transform_mode = vp9_read_literal( 2);
+        pProbSetup->transform_mode = (char)vp9_read_literal( 2);
```

Comment:
All this code must be changed to use concrete data types like int8_t, but this looks okay for now, until VP9 is done.